### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by Vov4ukz53**
This is a pagination component! 
When you hover the mouse over the button, I added an animation, please look at this solution. If you don't like it, we will discuss other the solution of mouse hovering.

### CODE REVIEW

File: src/common/Pagination/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {
+    Wrapper,
+    ArrowIcon,
+    Button,
+    TextButton,
```
Comment (olachrzan) [inline src/common/Pagination/index.js]: The name TextButton suggests a whole button with text, maybe call it ButtonText or just Text, which would suggest only text in the button. But this is my opinion 🙂

File: src/common/Pagination/styled.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        }
+    }
+    
+    ${({ next }) => next && css`
+        @media (min-width: ${({ theme }) => theme.breakpoints.large}) {
+            &:hover{
+                animation: ${move} 1s linear infinite reverse;
+            }
+        } 
+    `}
```
Comment (olachrzan) [inline src/common/Pagination/styled.js]: I would add an animation without a media query so that it appears every time, regardless of screen resolution

**Comment by olachrzan** [comment]
This animation is so cool :) But maybe add it just to the arrows so they move, not the whole button? What do you think?

File: src/common/Pagination/index.js (hunk 1 above)
Comment (Vov4ukz53) [reply src/common/Pagination/index.js]: Ok i will change it! Thankyou :)

File: src/common/Pagination/styled.js (hunk 2 above)
Comment (Vov4ukz53) [reply src/common/Pagination/styled.js]: I can remove media query, but when I pressed the button on the touchscreen, the button starts moving, can you check it in your browser,

**Comment by Vov4ukz53** [comment]
I wanted to add animation only to the arrow, but i don't know how to add an animation just on the arrow when hovering over the whole button ;)

File: src/common/Pagination/styled.js (hunk 2 above)
Comment (olachrzan) [reply src/common/Pagination/styled.js]: There is no hover on mobile devices! 😅 Leave the media as you have it, but maybe add :active pseudo-class

**Comment by Vov4ukz53** [comment]
I changed the animation to arrows and renamed TextButton to ButtonText. Also added active for the button.

**Review by olachrzan — APPROVED** [review]

**Pull request merged**